client/src/partials/patient_invoice/registry/registry.html

```diff
@@ -20,6 +20,12 @@
         >
         </bh-pdf-print>
       </div>
+
+
+      <div class="toolbar-item">
+        <bh-receipt-currency
```

Comment:
It's better practice to have close component tags, instead of having self-closing tags.  On Chrome, our target, it isn't a problem ... but it's bad practice.

For reference, https://github.com/angular/angular.js/issues/1953, https://github.com/angular/angular/issues/5563#issuecomment-161826319.